spec/fixtures/vcr/Import_BucketCake_data_into_Postgres/imports_data.yml

```diff
@@ -2,7 +2,7 @@
 http_interactions:
 - request:
     method: head
-    uri: https://ad2games-cake-data-test-fixtures.s3-eu-west-1.amazonaws.com/affiliates/latest.gz
+    uri: https://ad2games-cake-data-test-fixtures.s3.eu-west-1.amazonaws.com/affiliates/latest.gz
```

Comment:
Hum... seems this is the reason of why you needed to re-record vcr files. I wonder where this change of `s3-eu-west-1` to `s3.eu-west-1` came from.

Reply:
Hum indeed... 🤔

Reply:
The domain is probably set in the aws gem(s). Might be due to an update.